prisma/schema.prisma

```diff
@@ -0,0 +1,32 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+// Looking for ways to speed up your queries, or scale easily with your serverless or edge functions?
+// Try Prisma Accelerate: https://pris.ly/cli/accelerate-init
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Survey {
+  id   String @id @default(cuid())
+  Name String
+}
+
+model Question {
+  id       String    @id @default(cuid())
+  Question String
```

Comment:
if you use upper-case letters in variable names in this context, every time you refer to them you'll have to use double quotation marks